buildconfig/Jenkins/buildscript.bat

```diff
@@ -185,7 +185,7 @@ if not "%JOB_NAME%"=="%JOB_NAME:debug=%" (
   set VATES_OPT_VAL=ON
 )
 
-call cmake.exe -G "%CM_GENERATOR%" -DCMAKE_SYSTEM_VERSION=%SDK_VERS% -DCONSOLE=OFF -DENABLE_CPACK=ON -DMAKE_VATES=%VATES_OPT_VAL% -DParaView_DIR=%PARAVIEW_DIR% -DMANTID_DATA_STORE=!MANTID_DATA_STORE! -DUSE_PRECOMPILED_HEADERS=ON %PACKAGE_OPTS% ..
+call cmake.exe -G "%CM_GENERATOR%" -A %CM_ARCH% -DCMAKE_SYSTEM_VERSION=%SDK_VERS% -DCONSOLE=OFF -DENABLE_CPACK=ON -DMAKE_VATES=%VATES_OPT_VAL% -DParaView_DIR=%PARAVIEW_DIR% -DMANTID_DATA_STORE=!MANTID_DATA_STORE! -DUSE_PRECOMPILED_HEADERS=ON %PACKAGE_OPTS% ..
```

Comment:
https://social.msdn.microsoft.com/Forums/en-US/934aa55e-1641-4858-a7a5-a0c9bed4f85d/project-file-vctargetspatchvcxproj-cannot-find-visual-studio-2019-build-tools-even-though-i-have?forum=msbuild suggests that the `-A` needs to be quoted as: `"-A %CM_ARCH%"` or the solution won't build.

Reply:
`Win64` isn't listed in the [CMake 13.5 docs](https://cmake.org/cmake/help/git-stage/generator/Visual%20Studio%2016%202019.html) as an architecture; I've changed it to `x64`

Reply:
Good call. :+1: